source/connection-monitoring-and-pooling/connection-monitoring-and-pooling.rst

```diff
@@ -156,19 +168,24 @@ Connection Pool Members
--  **Single Track:** A Connection MUST limit itself to one request / response at a time. A Connection MUST NOT multiplex/pipeline requests to an Endpoint.
--  **Monotonically Increasing ID:** A Connection MUST have an ID number associated with it. Connection IDs within a Pool MUST be assigned in order of creation, starting at 1 and increasing by 1 for each new Connection.
+-  **Single Endpoint:** A `Connection <#connection>`_ MUST be associated with a single Endpoint. A `Connection <#connection>`_ MUST NOT be associated with multiple Endpoints.
+-  **Single Lifetime:** A `Connection <#connection>`_ MUST NOT be used after it is closed.
+-  **Single Owner:** A `Connection <#connection>`_ MUST belong to exactly one Pool, and MUST NOT be shared across multiple pools
+-  **Single Track:** A `Connection <#connection>`_ MUST limit itself to one request / response at a time. A `Connection <#connection>`_ MUST NOT multiplex/pipeline requests to an Endpoint.
+-  **Monotonically Increasing ID:** A `Connection <#connection>`_ MUST have an ID number associated with it. `Connection <#connection>`_ IDs within a Pool MUST be assigned in order of creation, starting at 1 and increasing by 1 for each new Connection.
 -  **Valid Connection:** A connection MUST NOT be checked out of the pool until it has successfully and fully completed a MongoDB Handshake and Authentication as specified in the `Handshake <https://github.com/mongodb/specifications/blob/master/source/mongodb-handshake/handshake.rst>`__, `OP_COMPRESSED <https://github.com/mongodb/specifications/blob/master/source/compression/OP_COMPRESSED.rst>`__, and `Authentication <https://github.com/mongodb/specifications/blob/master/source/auth/auth.rst>`__ specifications.
--  **Perishable**: it is possible for a connection to become **Perished**. A connection is considered perished if any of the following are true:
+-  **Perishable**: it is possible for a `Connection <#connection>`_ to become **Perished**. A `Connection <#connection>`_ is considered perished if any of the following are true:
 
-   -  **Stale:** The connection's generation does not match the generation of the parent pool
-   -  **Idle:** The connection is currently available and readyToUse, and has been for longer than **maxIdleTimeMS**.
-   -  **Errored:** The connection has experienced an error that indicates the connection is no longer recommended for use. Examples include, but are not limited to:
+   - **Stale:** The `Connection <#connection>`_ 's generation does not
+      match the generation of the parent pool
+   - **Idle:** The `Connection <#connection>`_ is currently
+      "available" (as defined below) and has been for longer than **maxIdleTimeMS**.
+   - **Errored:** The `Connection <#connection>`_ has experienced an
+      error that indicates it is no longer recommended for
+      use. Examples include, but are not limited to:
```

Comment:
The formatting of Stale/Idle/Errored is a bit off. Can you fix it? See the rendered view: https://github.com/mongodb/specifications/blob/ee56397b89b2da14b9114d0d71f59edec694fbaf/source/connection-monitoring-and-pooling/connection-monitoring-and-pooling.rst#id2

Reply:
I think this might be a bug in GitHub's rst renderer? I can't seem to get this to format right.

Reply:
Okay looks like I got it. It was missing a space in the list of perished states.